sdk/python/kfp/dsl/_metadata.py

```diff
@@ -55,7 +55,7 @@ def __init__(self,
       name: str = '',
```

Comment:
Parameter name is probably required (remove `= ''`).

Reply:
Done.